src/components/views/login/PasswordLogin.js

```diff
@@ -186,6 +186,17 @@ class PasswordLogin extends React.Component {
 
         const loginField = this.renderLoginField(this.state.loginType);
 
+        let matrixIdText = '';
+        if (this.props.hsUrl) {
+            try {
+                const parsedHsUrl = new URL(this.props.hsUrl);
+                matrixIdText = _t('%(serverName)s Matrix ID', {serverName: parsedHsUrl.hostname});
+            } catch (e) {
+                console.log(e);
```

Comment:
Might be useful to have something grep-able in the log here, like "Failed to parse HS URL"

Reply:
oh, good sport - I actually didn't mean to log this at all as it's normal that this will happen if you pause for a bit mid-typing